content/developer/Views.adoc

```diff
@@ -163,4 +165,89 @@ most common ones to override are:
   alter this behaviour or to output anything after the main display. You
   usually want to call parent::display(); to ensure that the display
   code is run (unless, of course, you are adding your own display
-  logic). link:../views[↩]
\ No newline at end of file
+  logic). link:../views[↩]
```

Comment:
Please remove that weird character from the end of the sentence. I know you didn't put it there, it's a left-over artifact from the old wiki we had, but let's take this chance to clean it up. Thanks